seq/src/main.rs

```diff
@@ -0,0 +1,125 @@
+use std::process;
+
+use clap::{load_yaml, App, AppSettings};
+
+#[derive(Clone,Debug)]
```

Comment:
Since they are all small `Copy` types, you could also derive `Copy`

Reply:
Okie dokie